ibm/acctest/acctest.go

```diff
@@ -1696,3 +1713,45 @@ func TestAccPreCheckScc(t *testing.T) {
 		t.Fatal("IBMCLOUD_SCC_REPORT_ID missing. Set the environment variable IBMCLOUD_SCC_REPORT_ID with a VALID REPORT_ID")
 	}
 }
+
+func TestAccProviderFactories() map[string]func() (*schema.Provider, error) {
+	return map[string]func() (*schema.Provider, error){
+		ProviderName:          func() (*schema.Provider, error) { return provider.Provider(), nil },
+		ProviderNameAlternate: func() (*schema.Provider, error) { return provider.Provider(), nil },
+	}
+}
+
+func Region() string {
+	region, _ := schema.MultiEnvDefaultFunc([]string{"IC_REGION", "IBMCLOUD_REGION", "BM_REGION", "BLUEMIX_REGION"}, "us-south")()
```

Comment:
I believe this function is replacing `IcdDbRegion`. Previously  `IcdDbRegion = os.Getenv("ICD_DB_REGION")`.
I don't see that env var in this list. Are we good here?

Reply:
This function replaces `IcdDbRegion` and will use the same defaults the provider uses to configure region. 
see https://github.com/IBM-Cloud/terraform-provider-ibm/blob/master/ibm/provider/provider.go#L93